test/e2e/Dockerfile.tests

```diff
@@ -0,0 +1,7 @@
+FROM fedora:33
```

Comment:
was this an accidental git add?

Reply:
there is a way to automate it with actions, but I think that's a little overkill for this. I just think we need to put this in a different directory and perhaps call it something like Dockerfile.frr? Maybe /test/e2e/images/ ?

Reply:
Moved the file, dropped a comment about it's purpouse and left a comment over the image name saying it comes from this dockerfile.